Sources/Java2SwiftLib/JavaTranslator.swift

```diff
@@ -77,6 +77,7 @@ extension JavaTranslator {
   package static let defaultTranslatedClasses: [String: (swiftType: String, swiftModule: String?, isOptional: Bool)] = [
     "java.lang.Class": ("JavaClass", "JavaKit", true),
     "java.lang.String": ("String", "JavaKit", false),
+    "java.lang.Object": ("JavaObject", "JavaKit", true)
```

Comment:
I believe this change shouldn't be necessary. `java.lang.Object` is already listed in `Java2Swift.config` for the `JavaKit` target, so the only way this could be needed is if something is missing a JavaKit dependency.